server/src/test/java/org/elasticsearch/monitor/fs/FsHealthServiceTests.java

```diff
@@ -238,7 +237,7 @@ private static class FileSystemIOExceptionProvider extends FilterFileSystemProvi
         AtomicBoolean injectIOException = new AtomicBoolean();
         AtomicInteger injectedPaths = new AtomicInteger();
 
-        private String pathPrefix = "/";
+        private String pathPrefix;
```

Comment:
Looks like this field could be final and moved into the ctor, since it is always only called once for each test-run.